LICENSE

```diff
@@ -199,48 +199,3 @@
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.
-
-
-Apache Software Foundation License 2.0
---------------------------------------
-java/fury-core/src/main/java/io/fury/util/Platform.java is modified from https://github.com/apache/spark/blob/master/common/unsafe/src/main/java/org/apache/spark/unsafe/Platform.java
```

Comment:
I'm not sure whether we can remove those projects like https://github.com/ray-project/ray or https://github.com/apache/arrow and https://github.com/apache/spark/blob/master/LICENSE all have this part.

maybe we can simplify this instead of removing this part

Reply:
From the Apache License 2.0 requirement, it writes:

> 1. You must give any other recipients of the Work or Derivative Works a copy of this License; and
> 2. You must cause any modified files to carry prominent notices stating that You changed the files; and
> 3. You must retain, in the Source form of any Derivative Works that You distribute, all copyright, patent, trademark, and attribution notices from the Source form of the Work, excluding those notices that do not pertain to any part of the Derivative Works; and

1 is met by we convey the LICENSE file, which happens to be Apache License 2.0.

2 is met by we comment on each copied and modified files.

3. is met by we retain the copyright header.

These are all the license required. Other projects can have their own conventional way, but it's not a requirement.

Reply:
I see, thanks for you explaintion